### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by racerpeter**
@diclophis

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -81,19 +81,18 @@ changestream {
...
-    sns {
-      // Optionally interpolate database name and/or table name in topic, for example:
-      //   "{database}-firehose"
-      //   "{tableName}-firehose"
-      //   "{database}.{tableName}-firehose"
-      // Note: String interpolation is only supported in SNS at this time
-      topic = "firehose"
-    }
+    // The SQS queue to which events should be sent. If the queue does not exist, it will be created.
+    sqs.queue = "firehose"
```
Comment (diclophis) [inline src/main/resources/application.conf]: I think we want to use values `ENV` for any type of configuration/naming of things, 12 factor style

File: src/main/scala/changestream/ChangestreamSupervisorStrategy.scala [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package changestream
+
+import akka.actor.SupervisorStrategy.{Escalate, Restart}
+import akka.actor.{OneForOneStrategy, SupervisorStrategy, SupervisorStrategyConfigurator}
+import com.github.mauricio.async.db.mysql.exceptions.MySQLException
+import scala.language.postfixOps
+import scala.concurrent.duration._
+
+final class ChangestreamSupervisorStrategy extends SupervisorStrategyConfigurator {
+  override def create(): SupervisorStrategy = OneForOneStrategy(loggingEnabled = true, maxNrOfRetries = 30, withinTimeRange = 1 minute) {
```
Comment (diclophis) [inline src/main/scala/changestream/ChangestreamSupervisorStrategy.scala]: lets load the retry options from `ENV`

File: src/main/resources/application.conf (hunk 1 above)
Comment (racerpeter) [reply src/main/resources/application.conf]: Yes, you'll notice that all values in application.conf can be overridden via env below (the `${?ENVNAME}` syntax

File: src/main/scala/changestream/ChangestreamSupervisorStrategy.scala (hunk 2 above)
Comment (racerpeter) [reply src/main/scala/changestream/ChangestreamSupervisorStrategy.scala]: Makes sense

Comment (racerpeter) [reply src/main/scala/changestream/ChangestreamSupervisorStrategy.scala]: I can't find an easy way to do this-- I'm going to suggest that we leave these values static for the time being. I suspect we may not need to adjust them at all once we have a good set of baseline values.

**Comment by racerpeter** [comment]
Updated PR with the addition of a pretty-print option for emitting to S3 in an Athena compatible way.

@diclophis can you take another look?

**Pull request merged**